<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qosmDialogBase</class>
 <widget class="QDialog" name="qosmDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QOSM Layer Properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select map type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maptypeSpinner"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="customUrl">
       <property name="placeholderText">
        <string>http://a.tile.opencyclemap.org/cycle/${z}/${x}/${y}.png</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addCustomType">
       <property name="text">
        <string>Add Custom Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="autorefresh">
       <property name="text">
        <string>Auto refresh map when extent changes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadTileCache">
       <property name="text">
        <string>Download Tile Cache...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="hasMaxZoom">
         <property name="text">
          <string>Max Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxZoom">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>22</number>
         </property>
         <property name="value">
          <number>19</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="hasFixedZoom">
         <property name="text">
          <string>Fixed Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fixedZoom">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>22</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="statusText">
     <property name="text">
      <string>Loaded %s of %s tiles at zoom level %s (automatic zoom would be %s).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>qosmDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>qosmDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasFixedZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixedZoom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hasMaxZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxZoom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
